File: common/src/main/java/com/zhongjh/common/entity/MediaExtraInfo.java
package com.zhongjh.common.entity;

/**
 * 多媒体参数类
 *
 * @author zhongjh
 * @date 2022/2/08
 */
public class MediaExtraInfo {

    private int width;
    private int height;
    private long duration;

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public long getDuration() {
        return duration;
    }

    public void setDuration(long duration) {
        this.duration = duration;
    }
}

File: common/src/main/java/com/zhongjh/common/entity/MediaExtraInfo.kt
package com.zhongjh.common.entity

/**
 * 多媒体参数类
 *
 * @author zhongjh
 * @date 2022/2/08
 */
class MediaExtraInfo {
    var width: Int = 0
    var height: Int = 0
    var duration: Long = 0
    /**
     * 具体类型，jpg,png,mp3等等
     * {@link MimeType }
     */
    var mimeType: String? = null
}